Documentation/coding-guidelines/interop-guidelines.md

```diff
@@ -287,4 +287,10 @@ one flavor's ABI and the shim's ABI.
     turns out there are many cases where being strictly 1:1 isn't practical.
 - Stick to data types which are guaranteed not to vary in size across flavors.
   - e.g. use int32_t, int64_t from stdint.h and not int, long.
-
+  - Use char* for ASCII or UTF-8 strings and uint8_t* for byte buffers.
```

Comment:
Nit: the "e.g." should be at the end of the previous sentence rather than at the beginning of this example line (since there are multiple example lines now)